docs/rfcs/20220209_limit_connections_rate.md

```diff
@@ -0,0 +1,91 @@
+- Feature Name: Ability to limit connection creation rate per shard
+- Status: in-progress
+- Start Date: 2022-02-09
+- Authors: Vadim Plakhtinskiy
+- Issue: 
+
+# Executive Summary
+
+- Set a limit on the rate with which the shard accepts new connections
+- Set a limit on connection creation rate per IP
+
+
+Similar to KIP-612, adapted to Redpanda's thread per core model
+
+## What is being proposed
```

Comment:
Can we also export metrics per broker? So that we know when client connections are being delayed because a given IP is being rate limited.

Reply:
Oh, thanks! I did not think about metrics